src/rpdk/guard_rail/rule_library/tags/schema-linter-core-tagging-rules.guard

```diff
@@ -137,7 +137,7 @@ rule ensure_property_tags_exists_v2 when tagging exists {
         tagging.permissions exists
         <<
         {
-            "result": "WARNING",
+            "result": "NON_COMPLIANT",
```

Comment:
Do we have documentation for our uses so that when they run this locally and it starts failing they know how to run it without these tests?

Reply:
Yes we have documented on how to run contract test locally with lintercheck exception